### PR TITLE
Improve performance of event reconnecting

### PR DESCRIPTION
**Pull request by almarklein**
In one particular use-case, this significantly improves performance.
@ivoflipse @korijn

### CODE REVIEW

File: flexx/event/_handler.py [hunk 1]
```diff
@@ -375,22 +375,40 @@ def _connect_to_event(self, index):
...
+        # Prepare disconnecting
+        old_objects = connection.objects  # (ob, type) tuples
+        connection.objects = []
+        
         # Obtain root object and setup connections
         ob = self._ob1()
         if ob is not None:
             self._seek_event_object(index, connection.parts, ob)
-
+        new_objects = connection.objects
```
Comment (Korijn) [inline flexx/event/_handler.py]: if `connection.objects` returns a reference, your code might not do what you think it does

Comment (almarklein) [reply flexx/event/_handler.py]: That's why there is a `connection.objects = []` a few lines above :)  (it's a simple attribute, not a property)

Comment (Korijn) [reply flexx/event/_handler.py]: okay.

File: flexx/event/tests/test_handlers.py [hunk 2]
```diff
@@ -581,4 +581,106 @@ def bar(self, *events):
...
+        f.bars = [Bar(), Bar()]
+    #
+    assert len(triggers) == 0
+    assert len(disconnects) == 0
+    
+    # Change values of bar.spam
+    with event.loop:
+        f.bars[0].spam = 7
+        f.bars[1].spam = 42
+    #
```
[the remark this answers is not in the document]
Comment (almarklein) [reply flexx/event/tests/test_handlers.py]: Like whitespace, but still keeping the block together. Sorry for the weird style :)

**Pull request merged**